src/osd/ReplicatedPG.cc

```diff
@@ -10477,6 +10477,24 @@ void ReplicatedPG::agent_clear()
   agent_state.reset(NULL);
 }
 
+bool ReplicatedPG::obccache_indicate_flush_evict(hobject_t& o)
+{
+  if (!object_contexts.lookup(o)) {
+    return false;
+  }
+  uint64_t divisor = pool.info.get_pg_num_divisor(info.pgid.pgid);
+  assert(divisor > 0);
+  uint64_t max_dirty_objects = (uint64_t)(pool.info.target_max_objects *
+    (float)pool.info.cache_target_dirty_ratio_micro / 1000000.0);
```

Comment:
* 1000000 / ..._micro (and avoid the floating point arithmetic?).  should be no danger of overflow here.